shared/cfg/machines.cfg

```diff
@@ -29,6 +29,13 @@ variants:
         only ppc64, ppc64le
         machine_type = pseries
         # Currently only supports standard VGA
+        # Make sure below param is set 'yes' to run a
+        # power8(compat mode) on power9 host
+        # power9_compatfixup = "no"
+        # Ref: https://github.com/torvalds/linux/commit/6d6ab940dc8b1c84fc86195c0f15a82ef282c8a3
+        # Make sure below param is set 'yes' to restore(enable)
+        # SMT after test
+        # restore_smt = "no"
```

Comment:
can we use same param to enable SMT after test instead using new param ?

Reply:
@balamuruhans Separate param is needed as we need to control across the tests, for example, set in first test and reset at last test of a suite...